torch/_dynamo/eval_frame.py

```diff
@@ -311,6 +311,15 @@ def get_compiler_fn(compiler_fn):
 def lookup_backend(compiler_fn):
     """Expand backend strings to functions"""
     if compiler_fn == "inductor":
+        if (
+            torch.backends.cuda.matmul.allow_tf32 is False
+            and torch.cuda.get_device_capability() >= (7, 0)
```

Comment:
(8, 0), tf32 is available on Ampere+ only, you shouldn't be getting a warning on V100.